src/reducers/reducers.test.js

```diff
@@ -301,6 +302,17 @@ describe('Reducer', () => {
     });
   });
 
+  describe('TOGGLE_CODE', () => {
+    it('should toggle the value of hoveredParameters', () => {
+      const newState = reducer(mockState.animals, {
+        type: TOGGLE_CODE,
+        visible: true,
+      });
+      expect(mockState.animals.visible.code).toBe(false);
```

Comment:
This test is not needed / relevant to what we are testing - please delete

Reply:
@studioswong  -- in our 1:1 the other day, you highlighted that this unrelated test was missing and it should added  as well. I personally think it should be in a new PR because it is unrelated but the consensus last time was to do everything int hsi PR.

Reply:
Hi Rashida, there seems to be some misunderstand - I am not referring for you to delete this entire test, but only this line (`expect(mockState.animals.visible.code).toBe(false);`) as it is not testing anything relevant to this test.

Reply:
thanks Susanna for the clarification. It wasn't clear in the initial comment. I will fix this.